Sources/MapboxCoreNavigation/EventDetails.swift

```diff
@@ -4,17 +4,18 @@ import Polyline
 import UIKit
 import AVFoundation
 import MapboxDirections
+import MapboxMobileEvents
 
 protocol EventDetails: Encodable {
-    var event: String? { get }
+    var event: String? { get set }
     var created: Date { get }
     var sessionIdentifier: String { get }
 }
 
 struct PerformanceEventDetails: EventDetails {
-    let event: String?
     let created: Date
     let sessionIdentifier: String
+    var event: String?
```

Comment:
This is because of the changes in `EventDetails`, this property is now settable too. `NavigationEventsManager` now works with a protocol instead of a concrete type.